docs/Options-in-Constructor.md

```diff
@@ -305,7 +306,7 @@ Complex HTML markup will be rendered as an image. For instance, this is suitable
 
 For each particular HTML question, you can override this general ("auto") behavior at the question level - by specifying a question's `renderAs` property (in a survey JSON definition). Set the property to "standard" or "image" as well (its default value is "auto").
```

Comment:
```suggestion
For each HTML question, you can override this general ("auto") behavior at the question level: specify a question's `renderAs` property (in a survey JSON definition); also set the property to "standard" or "image" (its default value is "auto").
```
This two sentences don't make sense to me. I don't really know what you are saying and how the second sentence relates to the first. Please rewrite.